cypress/support/index.js

```diff
@@ -16,6 +16,11 @@
 // Import commands.js using ES2015 syntax:
 import './commands';
 
+Cypress.on('uncaught:exception', (err, runnable) => {
```

Comment:
Instead of adding this here, could you create a custom command in the support/commands.js file? That's where other ignore uncaught exception commands are.

Reply:
done